src/components/MapCard/mapCard.module.scss

```diff
@@ -0,0 +1,77 @@
+$primary-color: #f16b67;
+ 
+.cardHeightNone {
```

Comment:
'None" implies 0 height, I think we should use the word 'default' in these cases